packages/squiggle-lang/src/rescript/Distributions/GenericDist.res

```diff
@@ -159,6 +119,89 @@ let toPointSet = (
   }
 }
 
+module Score = {
+  type scoreDistOrScalar = DistributionTypes.DistributionOperation.scoreDistOrScalar
+
+  type pointSet_ScoreDistOrScalar = D(PointSetTypes.pointSetDist) | S(float)
+  let argsMake = (
+    ~esti: scoreDistOrScalar,
+    ~answ: scoreDistOrScalar,
+    ~prior: option<scoreDistOrScalar>,
+  ): result<PointSetDist_Scoring.scoreArgs, error> => {
```

Comment:
I think I'd probably want to push most of this function into PointSetDist_Scoring.res. You still need to do ``toPointSetFn`` here, but the rest of it seems more general.

Reply:
I played around with this, but I don't like it. dependency cycles introduced seem like a lot of trouble.